gui/src/components/s3/iam-user-management.vue

```diff
@@ -647,6 +657,13 @@ export default class CortxIAMUserManagement extends Vue {
     this.$store.dispatch("systemConfig/hideLoader");
   }
 
+  public async checkPermissions() {
+    const routerApp: any = this;
```

Comment:
This is not routerApp. It is instance of vue. You can call it vueInstance instead of routerApp.

Reply:
Updated const name as vueInstance